testing/src/main/java/io/stargate/it/grpc/SimpleQueryTest.java

```diff
@@ -0,0 +1,31 @@
+package io.stargate.it.grpc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.grpc.ManagedChannel;
+import io.grpc.ManagedChannelBuilder;
+import io.stargate.it.BaseOsgiIntegrationTest;
+import io.stargate.it.storage.SkipIfBundleNotAvailable;
+import io.stargate.it.storage.StargateEnvironmentInfo;
+import io.stargate.proto.QueryOuterClass.Query;
+import io.stargate.proto.QueryOuterClass.Result;
+import io.stargate.proto.StargateGrpc;
+import org.junit.jupiter.api.Test;
+
+@SkipIfBundleNotAvailable(bundleName = "grpc")
```

Comment:
Use `./mvnw clean package -DskipTests -P default,grpc` to build locally. If `grpc` is not build then this annotation will skip the test automatically.